src/uproot/_util.py

```diff
@@ -457,7 +452,7 @@ def memory_size(data, error_message=None):
     if error_message is None:
         raise TypeError(
             "number of bytes or memory size string with units "
-            "(such as '100 MB') required, not {0}".format(repr(data))
+            "(such as '100 MB') required, not {}".format(repr(data))
```

Comment:
PyUpgrade doesn't simplify things like this, but it should, IMO:
```suggestion
            f"(such as '100 MB') required, not {data!r}"
```